src/edit/useFormKeyInfosEdit.js

```diff
@@ -1,126 +1,143 @@
+      return (
+        dirtyInputs[inputName] && !validInputs[inputName] && errorMessageDate
+      )
     }
-
-    function showErrorMessageKeyInfos(inputName) {
-        const errorMessageLocation = `Please fill in location`
-        const errorMessageDate = `Please choose a date`
-        const errorMessagePlayers = `Please fill in at least one player`
-        const errorMessageWinner = `Please fill in at least one winner`
-        const errorMessageShots = `Please fill in a number between 18 and 126`
-
-        if (inputName === 'location') {
-            return dirtyInputs[inputName] && !validInputs[inputName] && errorMessageLocation
-        }
-        if (inputName === 'date') {
-            return dirtyInputs[inputName] && !validInputs[inputName] && errorMessageDate
-        }
-        if (inputName === 'players') {
-            return dirtyInputs[inputName] && !validInputs[inputName] && errorMessagePlayers
-        }
-        if (inputName === 'winner' ) {
-            return dirtyInputs[inputName] && !validInputs[inputName] && errorMessageWinner
-        }
-        if (inputName === 'shots') {
-            return dirtyInputs[inputName] && !validInputs[inputName] && errorMessageShots
-        }
+    if (inputName === 'players') {
+      return (
+        dirtyInputs[inputName] && !validInputs[inputName] && errorMessagePlayers
+      )
     }
-
-    function handleSubmitKeyInfos(event) {
-        event.preventDefault()
-        updateTargetProfile(inputsKeyInfos, targetProfile)
-        resetDirtyInputs()
-        removeLocally(STORAGE_KEY)
-        updateVisibleForm('scores')
+    if (inputName === 'winner') {
+      return (
+        dirtyInputs[inputName] && !validInputs[inputName] && errorMessageWinner
+      )
     }
-
-    function handleCancelKeyInfos() {
-        resetDirtyInputs()
-        removeLocally(STORAGE_KEY)
-        showOverviewPage()
+    if (inputName === 'shots') {
+      return (
+        dirtyInputs[inputName] && !validInputs[inputName] && errorMessageShots
+      )
     }
+  }
 
-    function resetDirtyInputs() {
-        setDirtyInputs({
-            location: false,
-            date: false,
-            players: false,
-            winner: false,
-            shots: false,  
-        })
-    }
+  function handleSubmitKeyInfos(event) {
+    event.preventDefault()
+    updateTargetProfile(inputsKeyInfos, targetProfile)
+    resetDirtyInputs()
+    removeLocally(STORAGE_KEY)
+    updateVisibleForm('scores')
+  }
 
-    function showOverviewPage() {
-        history.push('/')
-    }
-}
\ No newline at end of file
+  function handleCancelKeyInfos() {
+    resetDirtyInputs()
+    removeLocally(STORAGE_KEY)
+    showOverviewPage()
+  }
+
+  function resetDirtyInputs() {
+    setDirtyInputs({
+      location: false,
+      date: false,
+      players: false,
+      winner: false,
+      shots: false,
+    })
+  }
+
+  function showOverviewPage() {
+    history.push('/')
+  }
+
+  function setDisabledState() {
```

Comment:
_(more context in useFormKeyInfosCreate.js)_
Maybe it's better to call it `changeDisabledToTrue` or something like that.